src/scene/data.ts

```diff
@@ -189,50 +189,57 @@ export async function exportToBackend(
   }
 }
 
-export async function importFromBackend(id: string | null, k: string | undefined) {
+export async function importFromBackend(
+  id: string | null,
+  k: string | undefined,
```

Comment:
can we use `key` to be more descriptive?

Reply:
if that's a `key`

Reply:
In the webcrypto api, "key" is the real key, but "k" is just the random piece of it. There's metadata around it. It tried to be consistent.

Reply:
Fair enough..